src/index.html

```diff
@@ -2,21 +2,30 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
+
     <meta
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     />
+
     <meta
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
+
     <title>Stop watch</title>
+
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
```

Comment:
The `href` attribute is linking to a SCSS file (`index.scss`). HTML cannot directly use SCSS files; they need to be compiled into CSS first. Ensure that the SCSS file is compiled to a CSS file and link to the resulting CSS file instead.